### PR TITLE
Fix example configuration

### PR DESCRIPTION
**Pull request by coyotebush**
The double-percent belongs first. Bump versions.

### CODE REVIEW

File: balboa-client-dispatcher/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ SBT:
  ```
  libraryDependencies ++= Seq(
    ...
-   "com.socrata" % "balboa-dispatcher-client" %% "[0.15, )",
+   "com.socrata" %% "balboa-dispatcher-client" % "0.16.0",
```
Comment (mhotan) [inline balboa-client-dispatcher/README.md]: Lets continue to use mathematical notation for patches.  This allows us to iterate on 0.16 patches.  The changes are small but usually substantial.

https://ant.apache.org/ivy/history/2.3.0/ivyfile/dependency.html#revision

File: balboa-client-kafka/README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ SBT:
  ```
  libraryDependencies ++= Seq(
    ...
-   "com.socrata" % "balboa-kafka-client" %% "0.14.0",
+   "com.socrata" %% "balboa-kafka-client" % "0.16.0",
```
Comment (mhotan) [inline balboa-client-kafka/README.md]: See above

File: balboa-service-kafka/README.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ This project is published as a maven repository and can be utilized and referenc
   ```
   libraryDependencies ++= Seq(
     ...
-    "com.socrata" % "balboa-kafka-service" %% "0.15.+",
+    "com.socrata" %% "balboa-kafka-service" % "0.16.0",
```
Comment (mhotan) [inline balboa-service-kafka/README.md]: See above

**Comment by mhotan** [comment]
Lets change this PR to change all the docs to use the same style version dependency.  That is Major.Minor.+ to consistently include patches.

**Comment by coyotebush** [comment]
Updated with the new project names from #26.

**Comment by mhotan** [comment]
:shipit:

**Pull request merged**